demo/flutter/copy_assets.sh

```diff
@@ -0,0 +1,17 @@
+if [ ! -d "./assets/contexts/android" ]
```

Comment:
may want to call the use the subfolder `./assets/models/android` instead so folks don't get confused between `.pv` and `.rhn` files